src/main/scala/scalismo/package.scala

```diff
@@ -38,6 +38,10 @@ package object scalismo {
       gc.SetScheduleTime(60, TimeUnit.SECONDS)
       gc.Start()
 
+      // we need to stop the garbage collection when the JVM shutdowns.
+      // Otherwise it prevents the JVM from shutting down correctly.
+      Runtime.getRuntime().addShutdownHook(new Thread { gc.Stop() })
```

Comment:
I'm not entirely sure if this is the correct way. To my understanding, this calls gc.Stop() in the **initializer** of the Thread, whereas it should actually be called when the Thread is **run**. Shouldn't this rather be

```
    Runtime.getRuntime.addShutdownHook(new Thread(){
      override def run(): Unit = gc.Stop()
    })
```

?